tests/benchmarks/cli/commands/test_init.py

```diff
@@ -1,12 +1,14 @@
 import os
 import shutil
+from contextlib import suppress
 
 
-def remove_dvc_root():
-    dvc_root = ".dvc"
-    if os.path.isdir(dvc_root):
+def _cleanup_dvc():
+    with suppress(FileNotFoundError):
         shutil.rmtree(".dvc")
+    with suppress(FileNotFoundError):
+        os.remove(".dvcignore")
 
 
 def test_init(bench_dvc, tmp_dir, scm):
-    bench_dvc("init", setup=remove_dvc_root, rounds=10, warmup_rounds=1)
+    bench_dvc("init", setup=_cleanup_dvc, rounds=10, warmup_rounds=1)
```

Comment:
How about instead of removing one-by-one we just remove everything in tmp_dir? So this could be future-proof too.